website/routes.py

```diff
@@ -94,6 +101,11 @@ def get_globals():
     else:
         request_login_url = request.url
 
+    try:
+        waffle_url = reverse('wafflejs')  # TODO: fix bug only effects Travis
```

Comment:
Another Travis only bug, it's likely because we're in a questionable request context here, some Flask elements, some DRF.

Reply:
I am wondering if we should just remove this wafflejs stuff altogether, along with the `wafflejs_url` included in the globals - navigating to `/_/wafflejs` throws an `AttributeError: 'WSGIRequest' object has no attribute 'user'` and I don't think we're actually using it.   

This was added to support WaffleJS - which allows using flags, switches, and samples in JavaScript.  Its usage is commented out here https://github.com/CenterForOpenScience/osf.io/blame/develop/website/templates/base.mako#L372.   (I'm the one that added this as part of https://github.com/CenterForOpenScience/osf.io/pull/7734 - original intent was to support as many different waffle use cases as I could at the time in an exploratory fashion, but we've not needed to go down this route).